Makefile

```diff
@@ -10,11 +10,17 @@ TGTS = graphql jsonschema docs shex owl csv graphql python
 #GEN_OPTS = --no-mergeimports
 GEN_OPTS =
 
-all: gen stage
+# We run all commands in the Pipenv.
+RUN = pipenv run
+
+all: pipenv-install gen stage
```

Comment:
It seems like other targets assume `pipenv-install` has been run, right? This could be expressed as a dependency. I think right now there is an assumption that Make will run the `pipenv-install` dependency here before the other two.

Reply:
Could maybe generate a marker file so that `pipenv-install` doesn't re-run every time (if you do add those dependency links).

Reply:
Fixed in fba27ff830c48970. I don't think we need a marker file, since `pipenv-install` should finish pretty quickly if everything is already installed.